Adding missing block names in checkout_cart_item_renderers.xml

jalogut opened the pull request:
Adding missing block names in `checkout_cart_item_renderers.xml` layout. Adding these names will allow to extend them using `referenceBlock` functionality.

jalogut committed:
Adding missing block names in checkout_cart_item_renderers.xml layout to be able to extend them using referenceBlock functionality

ishakhsuvarov commented:
Hi @jalogut 
Looks like the branch you have used as a base for PR is somewhat outdated, so the PR contains a merge conflict. Could you please sync with the latest `develop` branch and resolve conflict?
Thanks!

ishakhsuvarov committed:
magento/magento2#10319: Adding missing block names in checkout_cart_item_renderers.xml
 - Merge branch 'develop' of github.com:magento/magento2 into fix-missing-layout-block-names
 - Conflicts:
	app/code/Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml
